recipes/icu/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+if [ "$(uname)" == "Linux" ]; then
+    ./configure --prefix="$PREFIX"
```

Comment:
I'm confused (with regards to bit setting). The docs seem to suggest it is possible and simply provide two examples. We should try to set this for the relevant architectures explicitly even if it will try to figure it out on its own. In other words, explicit is better than implicit.

The commit message seems to suggest that they just cut out stuff to make the build faster. I agree with them (except for the tests and the static library). This will also save us on package size too, which should be desirable. As for the static library, we can check and see how this affects what is installed. Once we have tests though it will be easier to verify this doesn't have a negative effect.